mteb/tasks/Retrieval/law/GerDaLIRRetrieval.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class GerDaLIR(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="GerDaLIR",
```

Comment:
Can we rename this to distinguish from the other GerDaLIR?
Some models already have results for the other GerDaLIR which would otherwise be mixed with the current ones, see image. If there's no good name, we can also rename the older one and the results of those models, but I think it'd be better to rename this one, maybe GerDaLIRClean GerDaLIRSmall or something depending on what the exact difference is

<img width="848" alt="Screenshot 2024-04-04 at 10 31 41 AM" src="https://github.com/embeddings-benchmark/mteb/assets/62820084/60fd8945-a0d0-4125-b1c1-586cdaf54c95">

Reply:
GerDaLiRSmall sounds good for us! Could you help rename it in this code. I will update the name in the huggingface.

Reply:
Here is the new link of the dataset: https://huggingface.co/datasets/mteb/GerDaLIRSmall

Reply:
Great renamed everywhere & on the LB: https://huggingface.co/spaces/mteb/leaderboard